✨ IMPROVE: updating template

choldgraf opened the pull request:
This updates our template language to use the template variables in `hubs.yaml` and also to clean it up a bit and remove UoT language.

@yuvipanda what do you think?

Here's how it looks:

![image](https://user-images.githubusercontent.com/1839645/95524880-6da99f80-0987-11eb-9af6-32192b7d4e6a.png)

also opened up https://github.com/2i2c-org/pilot-hubs/issues/61

yuvipanda commented on templates/login.html:
```diff
@@ -17,13 +17,13 @@
     </a>
   <div id="operated-by">
-    Operated by <a href="https://2i2c.org/">2i2c.org</a> as part of the <a href="https://2i2c.org/pilot">2i2c Hubs for All pilot</a>.
+    Operated by <a href="https://2i2c.org/">2i2c.org</a> and <a href="https://www.cloudbank.org/">CloudBank</a> as part of the <a href="https://2i2c.org/pilot">2i2c Hubs for All pilot</a>.
```
Not all hubs are paid for by cloudbank credits, no? This should be a configurable option

choldgraf replied:
Yeah true - I guess we could add an `operated_by` and a `credits_by` field and insert them here? I'd imagine both would just take HTML that we can style as we like, though if we want a more complete solution we could design a deeper YAML structure to templatize, but that might be overengineering

yuvipanda commented on templates/login.html:
Same comment as earlier about cloudbank

choldgraf commented:
Addressed the above comments and added in configuration values for 3 roles:

- who operates the hub
- who funded the hub
- who designed the hub

What do you think about that @yuvipanda ?

Here's how it now looks:

![image](https://user-images.githubusercontent.com/1839645/95796526-f3885c00-0ca1-11eb-8d0e-3fa203f75af9.png)

yuvipanda commented:
Thanks a lot, @choldgraf!
